cmd/juju/status/status.go

```diff
@@ -421,5 +421,6 @@ func (c *statusCommand) Run(ctx *cmd.Context) error {
 }
 
 func (c *statusCommand) FormatTabular(writer io.Writer, value interface{}) error {
-	return FormatTabular(writer, c.color, value)
+	//enable coloring by default
```

Comment:
Maybe we introduce `--no-color` flag (with default value 'false') and use it there?

Reply:
Yes. I have reserved that for the next sub-task which is to add the `--no-color` flag.